Imagine/Cache/Resolver/PsrCacheResolver.php

```diff
@@ -0,0 +1,267 @@
+<?php
+
+/*
+ * This file is part of the `liip/LiipImagineBundle` project.
+ *
+ * (c) https://github.com/liip/LiipImagineBundle/graphs/contributors
+ *
+ * For the full copyright and license information, please view the LICENSE.md
+ * file that was distributed with this source code.
+ */
+
+namespace Liip\ImagineBundle\Imagine\Cache\Resolver;
+
+use Liip\ImagineBundle\Binary\BinaryInterface;
+use Psr\Cache\CacheItemInterface;
+use Psr\Cache\CacheItemPoolInterface;
+use function str_replace;
+use Symfony\Component\OptionsResolver\OptionsResolver;
+
+class PsrCacheResolver implements ResolverInterface
```

Comment:
```suggestion
final class PsrCacheResolver implements ResolverInterface
```

Implementing recommendation above